tests/controller_tests/test_filters.py

```diff
@@ -89,6 +89,51 @@ def test_get_funds(seeded_test_client):
     assert all(isinstance(fund["id"], str) for fund in response_json)
 
 
+class MockProgramme:
+    def __init__(self, fund_type_id):
+        self.fund_type_id = fund_type_id
+
+
+class FundTypeIdEnum:
+    TOWN_DEAL = "TD"
+    HIGH_STREET_FUND = "HS"
+
+
+def test_get_funds_alphabetically(seeded_test_client):
+    """
+    Test the function that retrieves funds in alphabetical order.
+
+    :Args:
+        seeded_test_client: A testing client with seeded data.
+
+    :Raises:
+        AssertionError: If the response to the GET request does not match the expected output.
+    """
+    read_org = entities.Organisation.query.first()
+
+    programme = entities.Programme(
+        programme_id="1",
+        programme_name="test programme",
+        fund_type_id="TD",
+        organisation_id=read_org.id,
+    )
+    db.session.add(programme)
+
+    programme = entities.Programme(
+        programme_id="2",
+        programme_name="test programme2",
+        fund_type_id="HS",
+        organisation_id=read_org.id,
+    )
+    db.session.add(programme)
```

Comment:
for future reference, when adding multiple entities you can add all with one db method call like:

`db.session.add_all((programme_1, programme_2, blah...))`

Reply:
Also, just 2 values is not the best test I think, might be worth adding a 3rd with a value that sits in-between the other 2, then you actually check more thoroughly it is re-ordered at query time. With 2 you could just be getting lucky on the order of 2 binary option

Reply:
understood implementing

Reply:
done

Reply:
nah, it's fine, I forgot it is set to enum on DB